tools/node_wrapper.ml

```diff
@@ -4,6 +4,8 @@ let extra_args_for_wasoo =
   ; "--stack-size=10000"
   ]
 
+let extra_args_for_jsoo = [ "--stack-size=3000" ]
```

Comment:
Can we have a comment explaining why we need this ?

Reply:
I have added that the double translation needed a larger stack. But I don't have a good explanation for that.

Reply:
I've pushed a change to only set stack-size with the with-effects-double-translation profile